source/renderer/app/components/staking/stake-pools/StakePoolsTable.tsx

```diff
@@ -1,134 +1,34 @@
-import React, { Component } from 'react';
+import React, { useEffect, useState, useCallback } from 'react';
 import { observer } from 'mobx-react';
-import { orderBy } from 'lodash';
 import classNames from 'classnames';
-import { defineMessages, intlShape, FormattedHTMLMessage } from 'react-intl';
-import { PopOver } from 'react-polymorph/lib/components/PopOver';
+import { injectIntl } from 'react-intl';
+import { useTable, useFlexLayout } from 'react-table';
+import List from 'react-virtualized/dist/commonjs/List';
+import WindowScroller from 'react-virtualized/dist/commonjs/WindowScroller';
+import AutoSizer from 'react-virtualized/dist/commonjs/AutoSizer';
```

Comment:
What is the reason of importing those components from internal paths instead of dependency root?
```suggestion
import { AutoSizer, List, WindowScroller } from 'react-virtualized';
```

Also there is corresponding `@types/react-virtualized` package availabe so it would be worth to install it.

Reply:
That's to reduce bundle size:
https://github.com/bvaughn/react-virtualized#getting-started

Reply:
That was the approach for older webpack versions. Starting from version 4 webpack is smarter and is able to do the tree shaking for such cases. They are also saying that in the readme.

Reply:
However I just noticed they removed the `sideEffects` property from `package.json` without updating the readme :/. It happened in the [v9.20.1](https://github.com/bvaughn/react-virtualized/blob/2e962d8f8aebc22cb1168a8d147bcaa1d27aa326/CHANGELOG.md#9201). Because of that webpack won't do the tree shaking by itself so unfortunately we have to import all coomponents individually..